### ISSUE TITLE
Shellcheck `SC2050` failing due to '${{ }}' -> '_____'

### ISSUE THREAD

**Issue by dwaynebailey** [rhysd/actionlint#45]
I get this error:

`SC2050:warning:2:56: This expression is constant. Did you forget the $ on a variable? [shellcheck]`

With code like this:

```yaml
      run: |
          if [[ "${{ github.event.inputs.chart_force_publish }}" = "true" ]]; then
            force="true"
          fi
```

So I assume it's balking because this is converted to:

```sh
          if [[ "_______________________" = "true" ]]; then
            force="true"
          fi
```

Can anyone see a workaround?

**Comment by rhysd** [comment]
Thank you for reporting this. I think it would be better to disable the rule.

**Comment by rhysd** [comment]
The fix was released at [v1.6.3](https://github.com/rhysd/actionlint/releases/tag/v1.6.3).
